Automattic/wp-calypso
# Domains: Add breadcrumbs to new transfer page

### Changes proposed in this Pull Request

This PR adds breadcrumbs to the new redesigned transfer page. They should look like the following screenshot:

<img width="832" alt="Screen Shot 2021-11-19 at 17 35 23" src="https://user-images.githubusercontent.com/5324818/142688708-a61d13da-b161-4068-af08-a12f30e0b924.png">

And in mobile view:

<img width="398" alt="Screen Shot 2021-11-19 at 17 40 50" src="https://user-images.githubusercontent.com/5324818/142689034-1312e31d-6600-4e19-a413-e76982d87b48.png">

This is part of the domain management redesign project described in pcYYhz-m2-p2.

### Testing instructions

- Build this branch locally or open the live Calypso link
- Go to "Upgrades > Domains"
- Select one of your registered domains
- If you're in the live Calypso link, please append ?flags=domains/transfers-redesign to your URL to enable the appropriate feature flag
- Select the "Transfer your domain" options

These should be checked:

- Ensure the transfer page contains breadcrumbs like shown in the screenshot above
- Please ensure the breadcrumb links take you to the correct pages - `Domains` should take you to the domains list page and `yourdomain.com` should take you to the specific domain's settings page
- Lastly, please also check the mobile view version and ensure the back link takes you to the domain settings page
